app/controllers/admin/dfc_product_imports_controller.rb

```diff
@@ -37,7 +37,8 @@ def index
       @count = imported.compact.count
     rescue Faraday::Error,
            Addressable::URI::InvalidURIError,
-           ActionController::ParameterMissing => e
+           ActionController::ParameterMissing,
+           Rack::OAuth2::Client::Error => e
       flash[:error] = e.message
       redirect_to admin_product_import_path
```

Comment:
Does the flash message persist or disappear automatically?

Reply:
In my testing I confirmed that error messages persist.
I remember this was intentional as part of the BUU update (other messages auto dismiss, but errors need to be dismissed by the user).